apps/webpack.js

```diff
@@ -123,6 +123,12 @@ var baseConfig = {
         'standaloneVideo',
         'locale-do-not-import.js'
       ),
+      '@cdo/aichat/locale': path.resolve(
+        __dirname,
+        'src',
+        'aichat',
+        'locale-do-not-import.js'
+      ),
```

Comment:
I'm actually curious whether this is needed.

Reply:
I get a webpack error without it.

Reply:
I think so long as we have lab-specific locale files, we will need these specific aliases.